crates/debug/src/transform/attr.rs

```diff
@@ -252,6 +266,26 @@ where
                 pending_di_refs.insert(current_scope_id, attr.name(), offset);
                 continue;
             }
+            AttributeValue::DebugAddrBase(offset) => {
+                if offset.0 != unit.addr_base.0 {
+                    bail!(
```

Comment:
return failure?

Reply:
Can we just ignore it? I was under impression this attribute only present for CU.

Reply:
Yes, we ignore it, thus the `continue`, but we also want to check if it is correctly extracted by `gimli` already.

Reply:
Done in 2e0b30a.